src/main/java/nextstep/domain/subway/PathType.java

```diff
@@ -1,8 +1,10 @@
 package nextstep.domain.subway;
 
+import nextstep.domain.Section;
+
 public enum PathType {
-    DISTANCE("DISTANCE"),
-    DURATION("DURATION");
+    DISTANCE("DISTANCE") {public Long getWeight(Section section){return section.getDistance();}},
```

Comment:
```suggestion
    DISTANCE("DISTANCE", Section::getDistance),
```

이런 형태로 구현할 수도 있습니다 😄